src/main/resources/db/changelog/changes/03-create-users-table.yaml

```
@@ -0,0 +1,45 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-table
+      author: mateacademy
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: firstName
```

did you test your implementation?
```suggestion
                  name: first_name
```

---

I did. After it had failed, I added another changeSet (04) for fixing the field names.